docs/validators/validators-btcli-guide.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Validator's Guide to `BTCLI`"
+---
+
+# Validator's Guide to `BTCLI`
+
+Validators evaluate miner performance, and post their evaluations to the blockchain. This page discusses considerations specific to validators when using `btcli`.
+
+For general coverage of `btcli` permissions and requirements, see: [Bittensor CLI: Permissions Guide](../btcli-permissions)
+
+See also: [Coldkey and Hotkey Workstation Security](../getting-started/coldkey-hotkey-security).
+
+:::tip 
+It is highly recommended to use a unique hotkey per subnet and rotate these when new subnets register. The reason being if a validator's single hotkey does get compromised, damage can be done by the attacker setting incorrect weights for miners or introducing a deadlock condition, effectively preventing normal operation.
```

Comment:
> rotate these when new subnets register
All hotkeys associated with subnets should be rotated if a new ones are registered? Creating a new hotkey per new subnet if not sufficient? Maybe a dumb question, but what is the reason. The rest makes sense to me

Reply:
Def not a dumb question. I'm not sure what I was thinking here to be honest haha